Dockerfile

```diff
@@ -42,14 +42,14 @@ RUN python3 -m pip install pip==20.2 flit==3.0.0
 RUN git clone https://github.com/mongodb/snooty-parser.git && \
 	cd snooty-parser && \
 	git fetch --tags && \
-	git checkout <TAG> && \
+	git checkout v0.9.6 && \
 	FLIT_ROOT_INSTALL=1 python3 -m flit install
 
 # install snooty front-end
 RUN git clone https://github.com/mongodb/snooty.git snooty
 RUN cd snooty && \
 	git fetch --all && \
-	git checkout <TAG> && \	
+	git checkout v0.9.8 && \	
```

Comment:
With these actions in place, we can deploy via bumping the tags in the Dockerfile via a pull request.

Reply:
I just want @sophstad and @i80and to see this and confirm we're comfortable with this change :)